### ISSUE TITLE
Emphasize that a Docker(-compatible) socket is required for container inputs

### ISSUE THREAD

**Issue by woodruffw** [trailofbits/it-depends#71]
This is a small documentation/UX thing: users running Podman-in-Podman or Docker-in-Docker will probably encounter failures, since `it-depends` talks directly to the Docker (or Docker-compatible) socket. 

We should document this as a limitation of the current implementation.

**Comment by chris-short** [comment]
Yes, please. Installed without realizing this. Docker is also not what I typically use to build containers.

**Comment by woodruffw** [comment]
Thanks for the feedback!

Did you install from a published package, or from the most recent `main`? We haven't published it yet, but Podman support should work as of https://github.com/trailofbits/it-depends/commit/67dc3f4a881ad3fb60b860b27197da2b52a20240.

If you're using another container build/run ecosystem, let us know and we'll look into supporting it.
